pom.xml

```diff
@@ -1495,6 +1495,16 @@
             <groupId>org.hdrhistogram</groupId>
             <version>${version.hdrhistogram}</version>
          </dependency>
+         <dependency>
+            <groupId>io.prometheus</groupId>
+            <artifactId>simpleclient_common</artifactId>
```

Comment:
can we drop this client? It is not used by Infinispan or Micrometer.

Reply:
this was added for the rest api code